From 098d9874ec249c8609f5a9010419b558926ce2c3 Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Fri, 21 Apr 2017 11:15:57 -0700
Subject: [PATCH 2374/2848] YARN-6360. Prevent FS state dump logger from
 cramming other log files (Contributed by Yufei Gu
 via Daniel Templeton)

(cherry picked from commit d49d1862a645b8c5d3eae95bf633bc82816f0168)
(cherry picked from commit bbd08bb7a4170d53557dfe4c26379c760b7ca12e)

Change-Id: I709a8203bf6a2aacb44c192252c6a39fa0c748ef
---
 .../hadoop-common/src/main/conf/log4j.properties   |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-common-project/hadoop-common/src/main/conf/log4j.properties b/hadoop-common-project/hadoop-common/src/main/conf/log4j.properties
index f297c71..6b7ee8f 100644
--- a/hadoop-common-project/hadoop-common/src/main/conf/log4j.properties
+++ b/hadoop-common-project/hadoop-common/src/main/conf/log4j.properties
@@ -271,6 +271,7 @@ log4j.appender.RMSUMMARY.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
 
 # Fair scheduler requests log on state dump
 log4j.logger.org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler.statedump=DEBUG,FSLOGGER
+log4j.additivity.org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler.statedump=false
 log4j.appender.FSLOGGER=org.apache.log4j.RollingFileAppender
 log4j.appender.FSLOGGER.File=${hadoop.log.dir}/fairscheduler-statedump.log
 log4j.appender.FSLOGGER.layout=org.apache.log4j.PatternLayout
-- 
1.7.9.5

